_opensearch/rest-api/index-apis/dangling-index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Dangling indices
+parent: index-apis
+grand_parent: REST API reference
+nav_order: 84
+---
+
+# Dangling indices API
+
+Dangling indices occur if any shards exist in a node's local directory that do not already exist in the cluster, after a node joins a cluster. Dangling indices can be listed, deleted, or imported.
+
+## Path and HTTP methods
+
+List dangling indices:
+
+```
+GET /_dangling
+```
+
+Import dangling index:
+
+```
+POST /_dangling/<index-uuid>
+```
+
+Delete dangling index:
+
+```
+DELETE /_dangling/<index-uuid>
+```
+
+## Path parameters
+
+Path parameters are required.
+
+Path parameter | Description
+:--- | :---
+index-uuid | UUID of index.
+
+## Query parameters
+
+Query parameters are optional.
+
+Query parameter | Data Type | Description
+:--- | :--- | :---
+accept_data_loss | boolean | Must be set to `true` for an `import` or `delete`, as Opensearch is unaware of where the dangling index data came from.
```

Comment:
Capitalize all items in the Data Type column. Remove the comma, and replace "as" with "because".